src/System.Private.CoreLib/shared/System/MemoryExtensions.cs

```diff
@@ -1606,5 +1606,111 @@ public static bool Overlaps<T>(this ReadOnlySpan<T> span, ReadOnlySpan<T> other,
                 value, comparer);
             return BinarySearch(span, comparable);
         }
+
+        /// <summary>
+        /// Sorts the elements in the entire <see cref="Span{T}" /> 
+        /// using the <see cref="IComparable" /> implementation of each 
```

Comment:
Nit: This is going to link to the non-generic IComparable rather than the generic one.  Same for other occurrences.